### PR TITLE
r/aws_connect_contact_flow_module

### PR DESCRIPTION
**Pull request by GlennChia**
### Community Note

* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for pull request followers and do not help prioritize the request

Relates #22340

Output from acceptance testing:

```
$ make testacc TESTS=TestAccConnectContactFlowModule_serial PKG=connect
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./internal/service/connect/... -v -count 1 -parallel 20 -run='TestAccConnectContactFlowModule_serial' -timeout 180m
=== RUN   TestAccConnectContactFlowModule_serial
=== RUN   TestAccConnectContactFlowModule_serial/basic
=== RUN   TestAccConnectContactFlowModule_serial/filename
=== RUN   TestAccConnectContactFlowModule_serial/disappears
--- PASS: TestAccConnectContactFlowModule_serial (459.82s)
    --- PASS: TestAccConnectContactFlowModule_serial/basic (185.60s)
    --- PASS: TestAccConnectContactFlowModule_serial/filename (167.39s)
    --- PASS: TestAccConnectContactFlowModule_serial/disappears (106.82s)
PASS
ok      github.com/hashicorp/terraform-provider-aws/internal/service/connect    467.348s

...
```

### CODE REVIEW

**Comment by GlennChia** [comment]
Re-ran the test cases after removing timeouts

```
make testacc TESTS=TestAccConnectContactFlowModule_serial PKG=connect
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./internal/service/connect/... -v -count 1 -parallel 20 -run='TestAccConnectContactFlowModule_serial' -timeout 180m
=== RUN   TestAccConnectContactFlowModule_serial
=== RUN   TestAccConnectContactFlowModule_serial/basic
=== RUN   TestAccConnectContactFlowModule_serial/filename
=== RUN   TestAccConnectContactFlowModule_serial/disappears
--- PASS: TestAccConnectContactFlowModule_serial (485.53s)
    --- PASS: TestAccConnectContactFlowModule_serial/basic (171.99s)
    --- PASS: TestAccConnectContactFlowModule_serial/filename (185.49s)
    --- PASS: TestAccConnectContactFlowModule_serial/disappears (128.04s)
PASS
ok      github.com/hashicorp/terraform-provider-aws/internal/service/connect    493.182s
```

**Review by AdamTylerLynch — APPROVED** [review]

**Review by AdamTylerLynch — APPROVED** [review]

**Comment by AdamTylerLynch** [comment]
LGTM 🚀

File: internal/service/connect/contact_flow_module.go [hunk 1]
```diff
@@ -0,0 +1,295 @@
...
+			},
+			"contact_flow_module_id": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"content": {
+				Type:             schema.TypeString,
+				Optional:         true,
+				Computed:         true,
+				ValidateFunc:     validation.StringIsJSON,
```
[the remark this answers is not in the document]
Comment (GlennChia) [reply internal/service/connect/contact_flow_module.go:48]: Yup, I believe the Amazon Connect Contact Flow resource is using that same validation currently

https://github.com/hashicorp/terraform-provider-aws/blob/f445f5e1980a8b4a6581b2964b8ba70b80330128/internal/service/connect/contact_flow.go#L48-L59

**Comment by breathingdust** [comment]
LGTM 🚀 

Verified Acceptance Tests in Commercial (us-west-2)

```console
make testacc TESTS=TestAccConnectContactFlowModule_serial PKG=connect
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./internal/service/connect/... -v -count 1 -parallel 20 -run='TestAccConnectContactFlowModule_serial' -timeout 180m
=== RUN   TestAccConnectContactFlowModule_serial
=== RUN   TestAccConnectContactFlowModule_serial/disappears
=== RUN   TestAccConnectContactFlowModule_serial/basic
=== RUN   TestAccConnectContactFlowModule_serial/filename
--- PASS: TestAccConnectContactFlowModule_serial (285.00s)
    --- PASS: TestAccConnectContactFlowModule_serial/disappears (85.13s)
    --- PASS: TestAccConnectContactFlowModule_serial/basic (100.29s)
    --- PASS: TestAccConnectContactFlowModule_serial/filename (99.59s)
PASS
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/connect	287.940s
```

Verified Acceptance Tests in GovCloud (us-gov-west-1)

```console
make testacc TESTS=TestAccConnectContactFlowModule_serial PKG=connect
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./internal/service/connect/... -v -count 1 -parallel 20 -run='TestAccConnectContactFlowModule_serial' -timeout 180m
=== RUN   TestAccConnectContactFlowModule_serial
=== RUN   TestAccConnectContactFlowModule_serial/basic
=== RUN   TestAccConnectContactFlowModule_serial/filename
=== RUN   TestAccConnectContactFlowModule_serial/disappears
--- PASS: TestAccConnectContactFlowModule_serial (279.23s)
    --- PASS: TestAccConnectContactFlowModule_serial/basic (128.13s)
    --- PASS: TestAccConnectContactFlowModule_serial/filename (82.02s)
    --- PASS: TestAccConnectContactFlowModule_serial/disappears (69.08s)
PASS
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/connect	282.530s
```

**Pull request merged**